.circleci/cb-publish-step-3-npm.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+# lerna has a bug (https://github.com/lerna/lerna/issues/1066) where failed publishes do not set the exit code properly
+# this causes the script to keep running even after failed publishes
+# this function forces failed publishes to exit on failure
+function lernaPublishExitOnFailure {
+  # exit on failure
+  set -e
+  # run lerna publish with the args that were passed to this function
+  # duplicate stdout to a temp file
+  # grep the temp file for the lerna err token and return exit 1 if found (-v option inverts grep exit code)
+  npx lerna publish "$@" | tee /tmp/publish-results && grep -qvz "lerna ERR!" < /tmp/publish-results
+}
```

Comment:
Can this function live in `.circleci/local_publish_helpers_codebuild.sh` instead of duplicating it?

Reply:
yes we can refactor after; I actually duplicated several cb-<name> files per Kamil's request to not touch existing .sh files